ci/vcpkg/ports.patch

```diff
@@ -174,23 +32,11 @@ index 2d6bba4d..0ac47887 100644
      PATCHES
          patch-relocatable-rpath.patch
          fix-aws-root.patch
+         fix-openssl3.patch
```

Comment:
This line was added in the json, unsure if this patch is still necessary

Reply:
This is patching the deprecated-declarations warning, does not work on manylinux2010 (gcc8) for some reason, overridden by -Wall maybe?